### PR TITLE
ROE-2300 Security analysis checks

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-2300

### Change description

make and pom files updated to include transitive dependency security analysis

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: pom.xml
Comment (markpit) [inline pom.xml:191]: looks like we'll need to include a suppress.xml file - see martin's PR - https://github.com/companieshouse/auth-code-notification/pull/44/files

Comment (mattch1) [reply pom.xml:191]: Suppress added.

Comment (markpit) [reply pom.xml:191]: looks good, just waiting for sonar checks to complete before approval

**Review by markpit — APPROVED** [review]

**Pull request merged**
